README.md

```diff
@@ -123,20 +147,58 @@ To initiate an upgrade process, a user has to update the `spec.image.tag` field
 
 After the `tag` field is updated, the StatefulSet will detect the version change and it will trigger the upgrade process.
 
-To detect whether a `ZookeeperCluster` upgrade is in progress or not, check the output of the command `kubectl get sts <name> -o yaml`. The output of this command contains the following entries
+To detect whether a `ZookeeperCluster` upgrade is in progress or not, check the output of the command `kubectl describe zookeepercluster`. Output of this command should contain the following entries:-
 
 ```
+$ kubectl describe zookeepercluster
+
 status:
-  collisionCount: 0
-  currentReplicas: 2
-  currentRevision: example-75d4b645c8
-  observedGeneration: 4
-  readyReplicas: 3
-  replicas: 3
-  updateRevision: example-5d9dfdb787
+Last Transition Time:    2020-05-18T10:25:12Z
+Last Update Time:        2020-05-18T10:25:12Z
+Message:                 0
+Reason:                  Updating Zookeeper
+Status:                  True
+Type:                    Upgrading
+```
+
+If the Status value for clustercondition type `Upgrading` is True, it signifies that the cluster is in Upgrading state.
+
+Additionally, The output for the `$ kubectl get zookeepercluster ` will also have the Desired version set to the Image Tag the zookeepercluster is trying to Upgrade to.
+
+```
+$ kubectl get zookeepercluster
+
+NAME         REPLICAS   READY REPLICAS   VERSION   DESIRED VERSION   INTERNAL ENDPOINT     EXTERNAL ENDPOINT   AGE
+example        3          3                0.2.6     0.2.5             10.100.200.126:2181   N/A                 11m
```

Comment:
It would be better if you could demonstrate by using desired version `0.2.7` as we encourage users to upgrade not downgrade

Reply:
I have tried upgrade with 0.2.7 and have changed the version to 0.2.7 in the desired version as well.